docs/samples/Create Asset.md

```diff
@@ -0,0 +1,178 @@
+# Create Asset
+
+This document will showcase how to create an asset with the new management API.
+
+> Please note: Before running the examples the corresponding environment variables must be set.
+> How such an environment can be setup locally is documented in [chapter 1](#1-optional---local-setup).
+
+## Table of Content
+
+1. [Optional - Local Setup](#1-optional---local-setup)
+2. [Terminology](#2-terminology)
+3. [Values](#3-values)
+4. [Simple Asset](#4-simple-asset)
+5. [Custom Property Asset](#5-custom-property-asset)
+6. [Private Property Asset](#6-private-property-asset)
+7. [Complex Property Asset](#7-complex-property-asset)
+
+## 1. Optional - Local Setup
+
+## 2. Terminology
+| Connector           | New Protocol (DCAT Catalogs) | Description                                                   |
+|---------------------|------------------------------|---------------------------------------------------------------|
+| Asset Entry         | Asset                        | Contains the Asset ID and its Data Address.                   |
+| Contract Definition | Asset Entry / Dataset        | Contains an Asset () that is offered and covered by a Policy. |
+
+## 3. Values
+| Key      | Description |
+|----------|-------------|
+| @context |             |
+| @vocab   |             |
+| edc      |             |
+
+## 4. Simple Asset
+
+````json
+{
+  "https://w3id.org/edc/v0.0.1/ns/asset":{
```

Comment:
generally, you don't have to specify the namespace for every property, and the context could be defined once at the root:
```json
{
  "@context":{
    "@vocab":"https://w3id.org/edc/v0.0.1/ns/",
    "edc":"https://w3id.org/edc/v0.0.1/ns/"
  },
  "asset": {
    "@type": "DataAddress",
    "@id": "some-asset-id",
    "properties": {
        "foo": "bar"
    }
  },
  "dataAddress": {
    "@type": "DataAddress",
    "type": "test-type",
    "keyName": "test-key-name"
  }
}
```

Reply:
Moved the context to the root in all examples.